src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -95,7 +95,7 @@ public async Task SslStream_StreamToStream_Authentication_IncorrectServerName_Fa
                 Task t2 = server.AuthenticateAsServerAsync(certificate);
 
                 await Assert.ThrowsAsync<AuthenticationException>(() => t1);
-                await t2;
+                // Ignore outcome of t2. It can succeed or fail depending on timing.
```

Comment:
Presumably t2 should always complete, though?  If so:
```suggestion
                try
                {
                    await t;
                 }
                 catch
                 {
                     // Ignore outcome of t2. It can succeed or fail depending on timing.
                 }
```

Reply:
yes. I was thinking about adding try/catch block but I did not see good reason. Maybe I can add .TimeoutAfter(TestConfiguration.PassingTestTimeoutMilliseconds) to be sure.

Reply:
> did not see good reason

Whenever possible, we want to avoid work associated with one test running after that test has completed.

Reply:
yes, that would make sense to avoid stray Tasks without linkage to particular test.